PICMI_Python/particles.py

```diff
@@ -282,6 +282,77 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_FoilDistribution(_ClassWithInit):
+    """
+    describes a foil with optional exponential pre-plasma ramps (front lower y/back higher y)
```

Comment:
A suggestion to help with clarity. If anymore information could be added, it would be helpful.
```suggestion
    Describes a foil with optional exponential pre- and post-plasma ramps along the propagation direction.
```